.travis.yml

```diff
@@ -49,10 +49,12 @@ rvm:
 - 2.4
 jobs:
   include:
+#    - stage: test
+#      script: "export PATTERN='models mailers integration workers lib' && cd test && rm -rf $PATTERN && echo 'Running tests:' && ls && cd - && bundle exec rake parallel:test[5] && cd test && ./format-coverage.sh && cd -"
+#    - stage: test
+#      script: "export PATTERN='controllers' && cd test && rm -rf $PATTERN && echo 'Running tests:' && ls && cd - && bundle exec rake parallel:test[5] && cd test && ./format-coverage.sh && cd -"
     - stage: test
-      script: "export PATTERN='models mailers integration workers lib' && cd test && rm -rf $PATTERN && echo 'Running tests:' && ls && cd - && bundle exec rake parallel:test[5] && cd test && ./format-coverage.sh && cd -"
-    - stage: test
-      script: "export PATTERN='controllers' && cd test && rm -rf $PATTERN && echo 'Running tests:' && ls && cd - && bundle exec rake parallel:test[5] && cd test && ./format-coverage.sh && cd -"
+      script: "bundle exec ruby test/models/bot/alegre_test.rb && bundle exec ruby test/models/project_media_test.rb -n /falsity/"
```

Comment:
Please revert this change

Reply:
I think we discussed that tests are no longer necessary - joke, will revert!